app/renderer/components/preferences/payment/history.js

```diff
@@ -113,82 +113,103 @@ class HistoryFooter extends ImmutableComponent {
       reconcileDate: formattedTimeFromNow(timestamp)
     }
 
-    return <div className={css(styles.historyFooter)}>
-      <div className={css(styles.nextPayment)}>
+    return <div className={css(styles.flexAlignCenter, styles.historyFooter)}>
+      <div className={css(styles.historyFooter__nextPayment)}>
         <span data-l10n-id={l10nDataId} data-l10n-args={JSON.stringify(l10nDataArgs)} />
       </div>
+      {/* TODO: refactor button.less */}
       <Button l10nId='paymentHistoryOKText'
-        className={css(commonStyles.primaryButton)}
+        className='primaryButton'
```

Comment:
Sorry, I didn't read you comment in the commit. Make sense, so let's create an issue for this and track all things that need's to be done there.